packages/main/src/TableVirtualizer.ts

```diff
@@ -270,7 +274,7 @@ class TableVirtualizer extends UI5Element implements ITableFeature {
 			}
 		}
 
-		if (scrollTopChange) {
+		if (scrollTopChange && this._scrollContainer) {
```

Comment:
so you are saying that there is a table where the user can focus on a row and then press some keyboard keys to navigate but at that moment the scrollContainer dom node of the table might not exists?

even the this._table check of this method in the first 3 lines is not necessary but it is just there to make the TS happy because this._table is nullable and complains that it can be empty and i do not like adding unnecessary exclamation marks around.